crates/biome_js_analyze/src/assist/source/organize_imports.rs

```diff
@@ -1,143 +1,328 @@
+    ///         }
+    ///     }
+    /// }
     /// ```
     ///
-    /// In these cases, Biome will sort all these three imports, and it might happen that the order will **break** your application.
+    /// ```js,expect_diagnostic,use_options
+    /// import { Database } from "bun:sqlite";
+    /// import lib from "@any/lib";
+    /// import test from "node:test";
+    /// import path from "path";
+    /// ```
     ///
-    /// To avoid this, create a "group" of imports. You create a "group" by adding a **new line** to separate the groups.
+    /// Note that, `import lib from "@any/lib"` doesn't match any explicitly set groups.
+    /// Imports and exports that don't match any user-provided groups, always match the implicit group that appears after all other groups.
+    /// Thus the import is placed at the end.
     ///
-    /// By doing so, Biome will limit the sorting only to the import statements that belong to the same group:
+    /// Groups can also be created using a glob or a list of globs.
+    /// In the following example, we create two groups: one that gathers imports/exports starting with `@any/lib` except `@any/lib/speciaal` and the other that gathers imports/exports starting with `@/`.
     ///
-    /// ```js
-    /// // group 1, only these two files will be sorted
-    /// import "../styles/reset.css";
-    /// import "../styles/layout.css";
+    /// ```json,options
+    /// {
+    ///     "options": {
+    ///         "groups": [
+    ///             ["@any/lib", "@any/lib/**", "!@any/lib/special", "!@any/lib/special/**"],
+    ///             "@/**"
+    ///         ]
+    ///     }
+    /// }
+    /// ```
     ///
-    /// // group 2, only this one is sorted
-    /// import { Grid } from "../components/Grid.jsx";
+    /// ```js,expect_diagnostic,use_options
+    /// import lib from "@any/lib";
+    /// import aliased from "@/alias";
+    /// import path from "@any/lib/special";
+    /// import test from "@any/lib/path";
     /// ```
     ///
-    /// ```js
-    /// // group 1, the polyfill/shim
-    /// import "../polyfills/array/flatMap";
+    /// Note that `@any/lib` matches `@any/lib` but not `@any/lib/**`.
+    /// Conversely, `@any/lib/subpath` matches `@any/lib/**`, but not `@any/lib`.
+    /// Thus you have to specify the two patterns if you want to accept all imports/exports that starts with `@any/lib`.
+    /// The prefix `!` indicates an exception.
+    /// For more details on the supported glob patterns, see the dedicated section.
+    ///
+    /// For now, it is not possible to mix predefined group and globs inside a list globs.
+    /// This is a limitation that we are working to remove.
     ///
-    /// // group 2, the files that require the polyfill/shim
-    /// import { functionThatUsesFlatMap } from "./utils.js";
+    /// The import organizer also allows you to indicate you want to separate two groups with a blank line using the predefined string `:BLANK-LINE:` like demonstrated in the following example.
```

Comment:
```suggestion
    /// The action also allows the separation of two groups with a blank line using the predefined string `:BLANK-LINE:` as demonstrated in the following example.
```

Let's explain the example.

Reply:
Hah, I just wrote an opposite suggestion above to replace "the action" with "the import organizer" :sweat_smile: I think "the action" sounds very abstract and takes the flow out of the sentence because it's so abstract that it may not be obvious what action is being referred to.